src/casync-tool.c

```diff
@@ -967,12 +970,26 @@ static int verbose_print_done_make(CaSync *s) {
         return 1;
 }
 
+static int get_total_extract_time(uint64_t *ret)
+{
```

Comment:
coding style nitpick: we place the opening bracket of a function on the same line as the function name, see other functions

Reply:
indeed, this is fixed now.